src/GammaEquationSystem.C

```diff
@@ -499,5 +607,81 @@ GammaEquationSystem::predict_state()
   gammaNp1.modify_on_device();
 }
 
+//--------------------------------------------------------------------------
+//------------------------- normalize dwalldistdx vector -------------------
+//--------------------------------------------------------------------------
+void
+GammaEquationSystem::normalize_dwalldistdx()
+{ 
+  stk::mesh::MetaData & meta_data = realm_.meta_data();
+  
+  const int nDim = meta_data.spatial_dimension();
+  
+  VectorFieldType *dwalldistdx = dWallDistdx_;
+  double vmag = 0.0;
+  
+  printf("normalize_dwalldistdx\n");
```

Comment:
remove print statement?

Reply:
Print statement removed and change pushed.